packages/client/auth-core/src/contexts/authlogin.tsx

```diff
@@ -78,18 +77,23 @@ export function AuthProvider({ children, crossmint, setJwtToken, appearance }: A
         }
     }, [crossmint.jwt]);
 
+    const value = useMemo(() => ({ login, logout, jwt: crossmint.jwt }), [login, logout]);
```

Comment:
Should crossmint be a dep here?

Reply:
Good catch, added!

Reply:
forget to commit the change here?

Reply:
Also, why `useMemo` here? This doesn't seem v/ resource intensive operation.

Reply:
removed